### DISCUSSION TITLE
How can I run a berry file?

### DISCUSSION THREAD

**Discussion by GijsTimmers** [arendst/Tasmota#20691] [Q&A] [answered]
I must be missing something.

I saved a `test.be` file in my Tasmota UFS, containing only this line:

    print("Hello World")

I want to run this file from the Tasmota console / Berry console. The text in quotes should pop up.

I have no idea on how to do this.

**Comment by sfromis** [comment, marked as answer]
`load('test.be')`

Or from the Tasmota console, the `Br` command executes the argument in Berry:
`Br load('test.be')`

Any definitions (like vars, functions or classes) from the file becomes available in the Berry context.

Using `"` vs `'` is the same, my preference is the latter, easier on my keyboard....

**Reply by GijsTimmers** [reply]
Thank you very much.

**Comment by jeroenst** [comment]
You can also use autoexec.be to run your berry script at boot.
